src/main/webapp/index.xhtml

```diff
@@ -14,6 +14,7 @@
     </h:head>
     <h:body>
         
+        <h:outputText styleClass="log-out" value="#{LocalTradeList.customerId}"/>

```

Comment:
This renders an empty span element for me above the menu. 
Would this be better in the menu component? could go in the drop-down menu after "trades"?